src/components/ProAmm/ProAmmPoolInfo.tsx

```diff
@@ -14,9 +15,29 @@ import { MouseoverTooltip } from 'components/Tooltip'
 import { ELASTIC_BASE_FEE_UNIT } from 'constants/index'
 import useProAmmPoolInfo from 'hooks/useProAmmPoolInfo'
 import useTheme from 'hooks/useTheme'
+import { MEDIA_WIDTHS } from 'theme'
 import { shortenAddress } from 'utils'
 import { unwrappedToken } from 'utils/wrappedCurrency'
 
+const MoneyBagIcon = () => {
```

Comment:
can we reuse MoneyBagIcon?  they are the same.

Reply:
Fixed